### ISSUE TITLE
[BUG] ERROR: Exception in runner thread -> Net:: -> stoul

### ISSUE THREAD

**Issue by molszews** [aristocratos/btop#59]
Ubuntu 18.04 after running `btop` command it immediately quits with message:
`ERROR: Exception in runner thread -> Net:: -> stoul`

I've tried to set `net_iface` to `wlo1` but no luck, I can confirm files like `/sys/class/net/wlo1/statistics/tx_bytes` and `/sys/class/net/wlo1/statistics/rx_bytes` are present

**Info (please complete the following information):**
 - btop++ version:  1.0.11
 - Binary: static
 - Architecture: x86_64
 - Platform: Linux
 - (Linux) Kernel: 5.4.0-84-generic

**Additional context**

contents of `~/.config/btop/btop.log`

```
2021/09/30 (19:58:49) | ===> btop++ v.1.0.11
2021/09/30 (19:58:49) | DEBUG: Starting in DEBUG mode!
2021/09/30 (19:58:49) | INFO: Logger set to DEBUG
2021/09/30 (19:58:49) | DEBUG: Setting LC_ALL=en_US.UTF-8
2021/09/30 (19:58:49) | INFO: Running on /dev/pts/2
2021/09/30 (19:58:49) | ERROR: Exception in runner thread -> Net:: -> stoul
2021/09/30 (19:58:49) | INFO: Quitting! Runtime: 00:00:00
```

**GDB Backtrace**

If btop++ is crashing at start the following steps could be helpful:

1. run `gdb btop`

2. `r` to run, wait for crash and press enter

3. `bt` to get backtrace

4. Copy and paste the backtrace here:
```
Starting program: /usr/local/bin/btop 
ERROR: Exception in runner thread -> Net:: -> stoul
[Inferior 1 (process 26102) exited with code 01]
(gdb) bt
No stack.
```

**Comment by aristocratos** [comment]
@molszews 
What do you get if you run `cat /sys/class/net/wlo1/statistics/tx_bytes` and `cat /sys/class/net/wlo1/statistics/rx_bytes` ?

**Comment by molszews** [comment]
```
cat /sys/class/net/wlo1/statistics/tx_bytes
197713440

cat /sys/class/net/wlo1/statistics/rx_bytes
3303907903

ls -la /sys/class/net/wlo1/statistics/
-r--r--r-- 1 root root 4096 wrz 30 20:01 collisions
-r--r--r-- 1 root root 4096 wrz 30 20:01 multicast
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_bytes
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_compressed
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_crc_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_dropped
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_fifo_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_frame_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_length_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_missed_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_nohandler
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_over_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 rx_packets
-r--r--r-- 1 root root 4096 wrz 30 20:01 tx_aborted_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 tx_bytes
-r--r--r-- 1 root root 4096 wrz 30 20:01 tx_carrier_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 tx_compressed
-r--r--r-- 1 root root 4096 wrz 30 20:01 tx_dropped
-r--r--r-- 1 root root 4096 wrz 30 20:01 tx_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 tx_fifo_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 tx_heartbeat_errors
-r--r--r-- 1 root root 4096 wrz 30 20:01 tx_packets
-r--r--r-- 1 root root 4096 wrz 30 20:01 tx_window_errors

```

**Comment by aristocratos** [comment]
@molszews 
Would you mind posting the output of for `rx_bytes`and `tx_bytes` for your other interfaces, unless you got loads of them.

You can try the following binary which should fix the issue, but I don't know the underlying problem yet (why I ask for more output). 
[btop.zip](https://github.com/aristocratos/btop/files/7261962/btop.zip)

**Comment by molszews** [comment]
sorry, I've just done a reboot and it seems to work just fine now - I think we can close this ticket and reopen when I manage to reproduce it

**Comment by krzyk** [comment]
I got exactly same error:
```
ERROR: Exception in runner thread -> Net:: -> stoul
```

Not sure why, I just downloaded latest release (btop-1.0.11-linux-i686.tbz) unpacked and tried ./bin/btop

**Comment by aristocratos** [comment]
@krzyk 
Try the binary linked above.

**Comment by krzyk** [comment]
@aristocratos the binary works without any issues

**Comment by aristocratos** [comment]
@krzyk 
As I asked above, would you mind posting the output from `cat /sys/class/net/<interface>/statistics/tx_bytes` and `cat /sys/class/net/<interface>/statistics/tx_bytes`

Where `<interface>` is all the the interfaces found in `/sys/class/net`

**Comment by krzyk** [comment]
for tx I get (iface + space + result of cat):
```
docker0 0
enp0s31f6 0
lo 63372007
teql0 0
tun0 2144740
wlp0s20f3 12062537696
```

for rx:
```
docker0 0
enp0s31f6 0
lo 63374581
teql0 0
tun0 9121744
wlp0s20f3 7567316421
```

**Comment by aristocratos** [comment]
@krzyk 
Thanks, everything looks okay, so not sure why it's happening.

Anyways, the fix will be in v1.0.12 later today.

**Comment by aristocratos** [comment]
Fixed in [v1.0.12](https://github.com/aristocratos/btop/releases/tag/v1.0.12)
